gwcli/utils.py

```diff
@@ -537,7 +538,15 @@ def __getattr__(self, name):
             try:
                 self.data = request_method(*self.args, **self.kwargs)
             except requests.ConnectionError:
-                raise GatewayAPIError("Unable to connect to api endpoint @ {}".format(self.args[0]))
+                msg = ("Unable to connect to api endpoint @ "
+                       "{}".format(self.args[0]))
+                self.data = Response()
+                self.data.status_code = 500
+                self.data._content = '{{"message": "{}" }}'.format(msg)
```

Comment:
Is the double ```{{``` and ```}}``` some hidden magic that translates it back to a single bracket?

Reply:
yup

Reply:
D'oh -- from the ```format``` call